_posts/2018-03-11-user-testing-warehouse.md

```diff
@@ -0,0 +1,54 @@
+
+Having recently completed the pages for logged in users (account settings, manage projects, etc.) these pages seemed like the best candidates for our initial round of testing.
+
+The test I constructed was simple: asking users about their current use of PyPI, then requesting them to complete a series of tasks on the new management pages. For the test, I recruited users who were package owners or maintainers - i.e. people who had experienced the old user interface and who would understand my instructions.
+
+Over three weeks, I ran tests with six different users, with each test taking between 25 minutes and one hour; the duration depending on how much the test subject had to say.
+
+As expected, the tests highlighted a number of problems in the UI, which in turn prompted me to open or tag [20 issues](https://github.com/pypa/warehouse/issues?q=is%3Aissue+label%3A%22user+testing%22) in our issue tracker. The vast majority of these tickets are simple changes that have been tagged as [good first issues](https://github.com/pypa/warehouse/issues?q=is%3Aissue+label%3A%22good+first+issue%22) - which should in turn help with our efforts to outreach to new contributors.
+
+Several of the test participants also raised new ideas, for which I will open new tickets soon.
+
+A big thanks to Ernest W. Durbin III, Nicolas Dubois, Łukasz Langa, Russell Keith-Magee, Alex Chan and Eliot Berriot for participating in these tests.
+
+## Next Steps
+
+From this intial program it is clear that user testing can bring value to open source projects. The challenge is to continue to conduct such tests once our MOSS funding is depleted.
+
+Fortunately, the Python community is engaged with the Warehouse project, and, so far, we've had no problems recruiting test participants. Usability experts [recommend a minumum of 5 users per test](https://www.nngroup.com/articles/how-many-test-users/), so as long as this goodwill continues, I don't anticipate running out of test subjects.
+
+On the other hand, setting up and running user tests takes time; and it would be nice to have some help with this to lighten the workload (and increase our testing bus factor). For this reason, today I've put out a call to [train up to three people on how to conduct and run user tests](TODO). My hope is that this opportunity will be taken by students (or UX enthusiasts) as a way to build their experience by working on a live, widely used application.
+
+Should this program be successful, I'd like to start group testing the [project detail page](https://pypi.org/project/Flask/) in the coming weeks. This page is the most important on the site, and coincidentally, the page we've also received the most feedback about.
```

Comment:
Could you also say something about what you would explore if you had more funding? For instance, recruiting users for these tests whose experiences would tell us more along the accessibility/internationalization dimension?